main.cpp

```diff
@@ -68,6 +68,7 @@ int main()
 			1,											// Number of queue family indices
 			&ComputeQueueFamilyIndex					// List of queue family indices
 		};
+		auto vkBufferCreateInfo = static_cast<VkBufferCreateInfo>(BufferCreateInfo);
```

Comment:
Can you explicity use the type instead of auto here?

Reply:
Sure, but why? :)
(Seriously: I just needed a working test, I'm not going to put further work into this, and I just wanted to let you know what is needed to compile with an updated toolchain.)

Reply:
Also: https://isocpp.github.io/CppCoreGuidelines/CppCoreGuidelines#Res-auto